python/graphstorm/model/gnn.py

```diff
@@ -202,22 +230,40 @@ def __init__(self):
         self._optimizer = None
 
     def get_dense_params(self):
-        """retrieve the all dense layers' parameters as a parameter list.
+        """retrieve the all dense layers' parameters as a parameter list except for
+        language model related parameters.
 
-        TODO(zhengda) we don't need this. We only need to call self.parameters().
```

Comment:
actually, we should consider removing this method. why do we need this method? Pytorch's `parameters` can already do the same thing.

Reply:
When you call parameters(), parameters of language models will also be collected.